### PR TITLE
estimated date range edges are now exclusive

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: R/utils-json.R [hunk 1]
```diff
@@ -218,6 +219,12 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, po
   return(approvals_all)
 }
 
+subsetByMonth <- function(pts, onlyMonth) {
```
Comment (lindsayplatt) [inline R/utils-json.R:219]: doesn't this function already exist - https://github.com/USGS-R/repgen/blob/master/R/uvhydrograph-data.R#L165-L170? Looks like it needs to be pulled out into `utils-shared.R`

File: R/utils-json.R [hunk 2]
```diff
@@ -231,21 +238,12 @@ getApprovalDates <- function(data, chain_nm, approval){
...
-      format <- "Ymd"
-    } else {
-      format <- "Ymd HMOS z"
-    }
-    
-    time <- parse_date_time(x,format, tz=ts$reportMetadata$timezone,quiet = TRUE)
-    if(isDV) { #want to render DVs at noon, convert to string to add noon and convert back to POSIXct
-      time <- time + hours(12)
-    }
+    time=flexibleTimeParse(x, ts$reportMetadata$timezone)
```
Comment (lindsayplatt) [inline R/utils-json.R]: ooo I like this! Might want to follow the current convention of `<-` instead of `=` though

File: R/utils-shared.R [hunk 3]
```diff
@@ -95,13 +95,17 @@ isLogged <- function(all_data, ts_data, series){
 
 ############ used in dvhydrograph-data, correctionsataglance-data, fiveyeargwsum-data, uvhydrograph-data ############ 
 
-formatDates <- function(char_date, type=NA, isDV=FALSE){
-  if(isDV) {
-    format <- "%F"
-  } else {
-    format <- "%FT%T"
-  }
+formatDates <- function(char_date, type=NA){
```
Comment (lindsayplatt) [inline R/utils-shared.R:98]: :+1:

File: R/utils-json.R (hunk 1 above)
Comment (thongsav-usgs) [inline R/utils-json.R:219]: oops, yes, I meant to delete from UV hydro

**Pull request merged**